corehq/apps/app_manager/tests/__init__.py

```diff
@@ -1,44 +1,44 @@
 
 try:
-    from corehq.apps.app_manager.tests.test_app_manager import *
-    from corehq.apps.app_manager.tests.test_xml_parsing import *
-    from corehq.apps.app_manager.tests.test_xform_parsing import *
-    from corehq.apps.app_manager.tests.test_form_versioning import *
-    from corehq.apps.app_manager.tests.test_form_preparation_v2 import *
-    from corehq.apps.app_manager.tests.test_days_ago_migration import *
-    from corehq.apps.app_manager.tests.test_suite import *
-    from corehq.apps.app_manager.tests.test_advanced_suite import *
-    from corehq.apps.app_manager.tests.test_suite_formats import *
-    from corehq.apps.app_manager.tests.test_suite_regex import *
-    from corehq.apps.app_manager.tests.test_suite_shadow_module import *
-    from corehq.apps.app_manager.tests.test_suite_form_filter_errors import *
-    from corehq.apps.app_manager.tests.test_media_suite import *
-    from corehq.apps.app_manager.tests.test_profile import *
-    from corehq.apps.app_manager.tests.test_build_errors import *
-    from corehq.apps.app_manager.tests.test_bulk_ui_translation import *
-    from corehq.apps.app_manager.tests.test_views import *
-    from corehq.apps.app_manager.tests.test_util import *
-    from corehq.apps.app_manager.tests.test_commcare_settings import *
-    from corehq.apps.app_manager.tests.test_brief_view import *
-    from corehq.apps.app_manager.tests.test_xpath import *
-    from corehq.apps.app_manager.tests.test_bulk_app_translation import *
-    from corehq.apps.app_manager.tests.test_models import *
-    from corehq.apps.app_manager.tests.test_extension_case import *
-    from .test_location_xpath import *
+    from .test_broken_build import *
     from .test_get_questions import *
+    from .test_location_xpath import *
     from .test_repeater import *
-    from .test_broken_build import *
     from .test_translations import *
-    from corehq.apps.app_manager.tests.test_case_meta import *
-    from corehq.apps.app_manager.tests.test_schedule import *
+    from corehq.apps.app_manager.tests.test_advanced_suite import *
+    from corehq.apps.app_manager.tests.test_app_manager import *
+    from corehq.apps.app_manager.tests.test_build_errors import *
+    from corehq.apps.app_manager.tests.test_bulk_app_translation import *
+    from corehq.apps.app_manager.tests.test_bulk_ui_translation import *
     from corehq.apps.app_manager.tests.test_case_list_form import *
-    from corehq.apps.app_manager.tests.test_form_workflow import *
     from corehq.apps.app_manager.tests.test_case_list_lookup import *
+    from corehq.apps.app_manager.tests.test_case_meta import *
     from corehq.apps.app_manager.tests.test_child_module import *
-    from corehq.apps.app_manager.tests.test_report_config import *
+    from corehq.apps.app_manager.tests.test_commcare_settings import *
+    from corehq.apps.app_manager.tests.test_days_ago_migration import *
+    from corehq.apps.app_manager.tests.test_dbaccessors import *
```

Comment:
Sorry, I got OCD and sorted these. I just added this import and removed test_brief_view.